### DISCUSSION TITLE
(CLOSED)

### DISCUSSION THREAD

**Discussion by RandomComradee** [IceHacks/SurvivCheatInjector#940] [General]
CLOSED

**Comment by aidan-laursen** [comment]
Any idea where to change the website in the app? I've sifted through the app files and couldn't find anything to change. And opening it doesn't show any options for changing it either. using the mac vers btw

**Comment by humphreygaming** [comment]
One, it's not that simple. 

Two, the app is already compiled as a custom Chrome browser instance that hosts the extension. You can not change it.

**Comment by TheOther1211** [comment]
Its not even the same code the code is heavily changed

**Comment by Yametekudashai** [comment]
Woowooo relaunch the cheat in survev 🐸 ik you wont, just want to check if the thread will appear in the server

**Reply by Yametekudashai** [reply]
alr it appears
